app/components/file-browser/component.ts

```diff
@@ -170,12 +170,15 @@ export default class FileBrowser extends Component.extend({
         return items && items.length;
     }
 
-    @computed('canEdit', 'hasItems', 'filter')
+    @computed('canEdit', 'hasItems', 'showFilterInput', 'showRename')
     get clickable(this: FileBrowser) {
-        const cssClass = ['.dz-upload-button'];
+        let cssClass: string | string[] = '';
```

Comment:
You can just do `const cssClass: string[] = []` and `cssClass.push('.dz-upload-button')` below.

Reply:
That's what I did initially, unfortunately dropzone throws an error if passed an empty list.